src/middlewared/middlewared/plugins/iscsi_/extents.py

```diff
@@ -302,14 +306,12 @@ async def clean_type_and_path(self, data, schema_name, verrors):
                         'You need to specify a filepath not a directory'
                     )
 
-            await check_path_resides_within_volume(
-                verrors, self.middleware, f'{schema_name}.path', path
-            )
+            self.validate_path_resides_in_volume(verrors, f'{schema_name}.path', path)
```

Comment:
A synchronous function cannot call a coroutine so this will fail.

Reply:
I _think_ I've addressed all your concerns, @yocalebo ... but, please let me know if there are other gremlins 😛